### PR TITLE
Adding Makefile which will content - Build openstack-ansibleee-runner…

### PR DESCRIPTION
**Pull request by bshewale**
… image from openstack-ansibleee-operator/blob/main/Makefile

### CODE REVIEW

**Review by rebtoor — CHANGES_REQUESTED** [review]
This Makefile, as is, is almost useless here (this isn't an operator repo, this repo doesn't follow the operators lifecycle, there's no go code and so on...). We only need the runner related targets `docker-build-ee` and `docker-push-ee`.  Could you kindly fix? Thanks!

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+.PHONY: gowork
+gowork: ## Generate go.work file to support our multi module repository
+	test -f go.work || go work init
+	go work use .
+	go work use ./api
+
+.PHONY: operator-lint
+operator-lint: gowork ## Runs operator-lint
+	GOBIN=$(LOCALBIN) go install github.com/gibizer/operator-lint@latest
+	go vet -vettool=$(LOCALBIN)/operator-lint ./... ./api/...
```
Comment (rebtoor) [inline Makefile:184]: you can remove all these lines.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,184 @@
+##@ Build openstack-ansibleee-runner image
```
Comment (rebtoor) [inline Makefile:1]: Before the first target, add some useful variables that we lost in the last edit:
```

# openstack-ansibleee-runner image
EEIMG ?= quay.io/openstack-k8s-operators/openstack-ansibleee-runner

# edpm-ansible local repo
EDPM_LOCAL_REPO ?= ""

VERIFY_TLS ?= true


```

**Review by rebtoor — CHANGES_REQUESTED** [review]

**Comment by jpodivin** [comment]
@rebtoor I'd prefer not to do the makefile at all tbh. At this point it would be just two, exceedingly short, shell snippets and we can handle that in other ways, like with shell.

**Comment by rebtoor** [comment]
> @rebtoor I'd prefer not to do the makefile at all tbh. At this point it would be just two, exceedingly short, shell snippets and we can handle that in other ways, like with shell.

Yeah, probably you're right.

**Pull request closed without merging**